engine/trainer.py

```diff
@@ -284,6 +284,11 @@ def test(self):
             all_matching_stats = []
             if self.cfg.TEST.VORONOI_ON_MASK:
                 all_matching_stats_voronoi = []
+                
+        if self.cfg.TEST.MATCHING_VJI_PAI and self.cfg.PROBLEM.TYPE == 'INSTANCE_SEG' and self.cfg.DATA.TEST.LOAD_GT:
+            all_matching_stats = []
```

Comment:
Do not use the same variable name "all_matching_stats" and "all_matching_stats_voronoi" as the ones used above when self.cfg.TEST.MATCHING_STATS is True as if both are calculated the values will be mixed,